alembic/versions/167243c0e86c_add_address_fields_for_groups.py

```diff
@@ -0,0 +1,73 @@
+"""Add address fields for groups
+
+Revision ID: 167243c0e86c
+Revises: b6074f8ea4ab
+Create Date: 2017-07-12 22:34:43.889923
+
+"""
+
+
+# revision identifiers, used by Alembic.
+revision = '167243c0e86c'
+down_revision = 'b6074f8ea4ab'
```

Comment:
I don't think this revision is correct. I think it's supposed to be `71991162a59c`?